Gruntfile.coffee

```diff
@@ -1,3 +1,4 @@
+babel = require('rollup-plugin-babel');
```

Comment:
Since this is the only coffee script in the repo can we just switch this to be a normal js file for consistency